libs/shared/cms/src/lib/queries/relying-party/factories.ts

```diff
@@ -19,10 +19,31 @@ export const RelyingPartyResultFactory = (
 ) => ({
   clientId: faker.string.hexadecimal(),
   entrypoint: faker.string.hexadecimal(),
+  name: faker.string.sample(),
   EmailFirstPage: {
     headline: faker.string.sample(),
     description: faker.string.sample(),
-    submitButtonColorHex: faker.string.hexadecimal({ length: 6 }),
+    primaryButtonText: faker.string.sample()
+  },
+  SignupSetPasswordPage: {
+    headline: faker.string.sample(),
+    description: faker.string.sample(),
+    primaryButtonText: faker.string.sample(),
+  },
+  SignupConfirmCodePage: {
+    headline: faker.string.sample(),
+    description: faker.string.sample(),
+    primaryButtonText: faker.string.sample()
+  },
+  SignupConfirmedSyncPage: {
+    headline: faker.string.sample(),
+    description: faker.string.sample(),
+    primaryButtonText: faker.string.sample(),
+  },
+  shared: {
+    buttonColor: faker.color.rgb(),
+    logoUrl: faker.internet.url(),
+    logoAltText: faker.string.sample(),
   },
   ...override,
 });
```

Comment:
Is this for tests?  From the file path it's not obvious that it is.

Reply:
Yea these factories are use in tests.